demos/storybook/stories/drawer/with-different-variants.tsx

```diff
@@ -0,0 +1,21 @@
+import { Drawer, DrawerBody, DrawerHeader, DrawerNavGroup } from '@pxblue/react-components';
+import { boolean, select } from '@storybook/addon-knobs';
+import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
+import React from 'react';
+import { DrawerStoryContext } from './util';
+import { navGroupItems1 } from './with-basic-config';
+import { Menu } from '@material-ui/icons';
+
+export const withDifferentVariants = (context: DrawerStoryContext): StoryFnReactReturnType => (
+    <Drawer
```

Comment:
Might be more useful to show this in the context of a DrawerLayout so it has more context re: how it behaves with respect to other content.

Reply:
I would still like to see this in a DrawerLayout...it's hard to understand what the different variants are with the drawer moving around on the screen